### ISSUE TITLE
Accessibility: Unable to view the applied filters tooltip on dashboards

### ISSUE THREAD

**Issue by yousoph** [apache/superset#26216]
#### How to reproduce the bug

1. Create a dashboard
2. Set up some filters on the dashboard 
3. Save the dashboard
4. On the dashboard you created, apply a filter 
5. Use your keyboard to navigate on the dashboard 

### Expected results
User should be able to open the applied filters menu on charts that have a filter applied to see the filters list 
<img width="1499" alt="image" src="https://github.com/apache/superset/assets/10627051/5fc239b2-e41e-4a80-ada8-bbb2e5ca1a19">
![image](https://github.com/apache/superset/assets/10627051/f82cd64e-fab3-440f-8e52-5b7ff9b2c60b)

https://www.w3.org/TR/WCAG22/#keyboard

### Actual results

Unable to open the menu 

### Environment

(please complete the following information):

- browser type and version:
- superset version: `master`
- python version: `python --version`
- node.js version: `node -v`
- any feature flags active:

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by yousoph** [comment]
@eschutho has a draft for some work on keyboard navigation in dashboards, might be worth checking in with her to make sure none of the work will overlap or be duplicated

**Comment by yousoph** [comment]
https://github.com/apache/superset/pull/28015
